src/pages/chainlink-nodes/v1/using-forwarder.md

```diff
@@ -0,0 +1,353 @@
+- Allow the two Chainlink node EOAs to call the forwarder's [forward](/chainlink-nodes/contracts/forwarder#forward) function. Because the operator contract owns the forwarder contract, call [acceptAuthorizedReceivers](/chainlink-nodes/contracts/operator#acceptauthorizedreceivers) on the operator contract. Specify the forwarder contract address and the two Chainlink node EOAs as parameters. This call makes the operator contract accept ownership of the forwarder contract and authorizes the Chainlink node EOAs to call the forwarder contract by calling [setauthorizedsenders](/chainlink-nodes/contracts/receiver#setauthorizedsenders).
+
+### Whitelist the forwarder
+
+In the blockchain explorer, view the operator contract and call the `setAuthorizedSenders` method with the address of your forwarder contract. The parameter is an array. For example, `["0xA3f07D6773514480b918C2742b027b3acD9E44fA"]`. Metamask prompts you to confirm the transaction.
+
+<ClickToZoom src='/images/chainlink-nodes/node-operators/forwarder/operatorsetauthorizedsenders.jpg' />
+
+### Whitelist the Chainlink node EOAs
+
+In the blockchain explorer, view the operator contract and call the `acceptAuthorizedReceivers` method with the following parameters:
+
+- `targets`: Specify an array of forwarder addresses. For example, `["0xA3f07D6773514480b918C2742b027b3acD9E44fA"]`
+- `sender`: Specify an array with the two Chainlink node EOAs. For example, `["0x259c49E65644a020C2A642260a4ffB0CD862cb24","0x71a1Eb6534054E75F0D6fD0A3B0A336228DD5cFc"]`.
+
+Metamask prompts you to confirm the transaction.
+
+<ClickToZoom src='/images/chainlink-nodes/node-operators/forwarder/acceptAuthorizedReceivers.jpg' />
+
+## Activate the forwarder
+
+1. In the shell of your Chainlink node, enable the forwarder using the Chainlink CLI. Replace `forwarderAddress` with the forwarder address that was created by the factory. Replace `chainId` with the EVM chain ID. (`11155111` for Sepolia):
+
+   ```shell
+   chainlink forwarders track --address forwarderAddress --evmChainID chainId
+   ```
+
+   In this example, the command is:
+
+   ```shell
+   chainlink forwarders track --address 0xA3f07D6773514480b918C2742b027b3acD9E44fA --evmChainID 11155111
+   ```
+
+   ```shell
+   Forwarder created
+   ------------------------------------------------------
+   ID:         1
+   Address:    0xA3f07D6773514480b918C2742b027b3acD9E44fA
+   Chain ID:   11155111
+   Created At: 2023-03-02T11:41:43Zchainlink@22480bec8986
+   ```
+
+1. Exit the shell of your Chainlink node:
+
+   ```shell
+   exit
+   ```
+
+1. Stop your Chainlink node:
+
+   ```shell
+   docker stop chainlink && docker rm chainlink
+   ```
+
+1. Update your environment file. If you followed [Running a Chainlink Node locally](/chainlink-nodes/v1/running-a-chainlink-node) guide then add `ETH_USE_FORWARDERS=true` and `FEATURE_LOG_POLLER=true` to your environment file:
```

Comment:
We'll need to update this for TOML format later, but for now .env is fine. Just a reminder.

Reply:
yes indeed.